lib/cards/KCard.vue

```diff
@@ -436,23 +472,26 @@
       }
     },
     methods: {
-      onLinkFocus() {
+      onTitleFocus() {
         if (this.isSkeleton) {
           return;
         }
-        this.isLinkFocused = true;
+        this.isTitleFocused = true;
       },
-      onLinkBlur() {
-        this.isLinkFocused = false;
+      onTitleBlur() {
+        this.isTitleFocused = false;
       },
       onThumbnailError() {
         this.thumbnailError = true;
       },
-      navigate() {
-        if (this.isSkeleton) {
-          return;
+      clickHandler() {
+        /**
+         * Emitted when a card is clicked
+         */
+        this.$emit('click');
```

Comment:
Its usually useful to emit the event object here too. I have found myself many times updating some KDS components because I need the event object, so I think it doent hurt if we emit it too since the beginning.

Reply:
Ah definitely. Done.